### PR TITLE
新增about-appium/getting-started.md翻译

### PR DESCRIPTION
**Pull request by XiufanJi**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/cn/about-appium/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
+## 新手入门
+
+本篇文章将会带你了解并运行一个简单的appium案例以及向你介绍一些关于appium的基本概念。想要了解更多关于appium的详细信息，请查看[相关介绍](/docs/en/about-appium/intro.md)。
```
Comment (lihuazhang) [inline docs/cn/about-appium/getting-started.md]: 这里的相关介绍也是跳转到英文文档的，可以link去中文的文档吧？

File: docs/cn/about-appium/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+### 安装appium
+
+Appium可以通过以下两种方式中的任意一种进行安装：使用[NPM](https://npmjs.com)命令或者是下载[appium客户端](https://github.com/appium/appium-desktop)
+，一种基于桌面图形化的方式来登录appium服务。
+
+#### 使用NPM命令进行appium安装
+
+如果你想使用`npm install`命令进行appium的安装，hack操作或者是为appium做贡献，你都需要安装[Node.js和NPM](http://nodejs.org)（使用[nvm](https://github.com/creationix/nvm),[n](https://github.com/visionmedia/n)命令，或者是`brew install node`命令来安装node.js。请确认你没有使用`sudo`命令进行Node或者Appium的安装，否则运行时会出现问题）尽管Appium支持Node10及以上版本，我们仍然推荐使用最新稳定版。
```
Comment (lihuazhang) [inline docs/cn/about-appium/getting-started.md]: 如果你想使用`npm install`命令进行appium的安装，hack操作或者是为appium做贡献
建议改成

如果你想用 `npm install`命令安装appium，或者改造appium，或者向appium提交代码做贡献

File: docs/cn/about-appium/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+### 不同系统的驱动安装
+
+你有可能会使用Appium来进行不同应用的自动化操作，例如IOS应用或者是Android应用。而Appium“驱动”则提供了对于不同平台的自动化操作。下面列举的驱动类型可以使你在不同的平台都能使用自动化技术，并且每种驱动都有他们特定的安装要求。对于某个平台来说，驱动的安装要求与在该平台上进行app开发所需的环境要求大多一致。例如，如果你想使用Android驱动来进行Android应用的自动化操作，那么你所使用的的系统中应该设置好Android SDK.
+
+在某些情况下，对于你想进行自动化操作的平台，请确认你查看过对应驱动的文档，以便能正确进行安装：
+- [XCUITest 驱动](/docs/en/drivers/ios-xcuitest.md) (IOS和tvOS应用使用)
+- [Espresso 驱动](/docs/en/drivers/android-espresso.md) (Android应用使用)
+- [UiAutomator2 驱动](/docs/en/drivers/android-uiautomator2.md) (Android应用使用)
+- [Windows 驱动](/docs/en/drivers/windows.md) (Windows操作系统桌面应用使用)
+- [Mac 驱动](/docs/en/drivers/mac.md) (Mac操作系统桌面应用使用)
```
Comment (lihuazhang) [inline docs/cn/about-appium/getting-started.md]: driver 不需要翻译，翻译了反而难理解。
iOS 不是 IOS，这个要专业一点。

File: docs/cn/about-appium/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+你可以使用`appium-doctor`命令来确认安装Appium所需的依赖是否都已满足要求。使用`npm install -g appium-doctor`命令来进行上述命令的安装，安装完成后，运行`appium-doctor`命令确认Appium所需的依赖都已正确安装，该命令还支持与`--ios`或者 `--android`选项一起搭配使用。
+
+### Appium客户端
+
+即使当上述所需条件都已满足，Appium也只是一个HTTP服务器而已。它需要与客户端进行连接，并由客户端发出指令告诉它应该开启哪种会话，还有一旦会话启动成功后该进行哪些自动化操作。这就意味着你永远不能只单独使用Appium本身，你需要将它与某一个客户端一起搭配使用（或者，如果你具有探索精神，可以使用cURL的方式进行尝试）
+
+幸运的是，Appium与[Selenium](http://www.seleniumhq.org/)使用同一种协议，称为WebDriver协议。也许在你使用的系统中已经安装了多个客户端，但你只需要搭配任意一种标准Selenium客户端便足够运行Appium进行多种操作，特别是当你使用Appium来进行移动平台上的浏览器测试的时候。
+
+尽管如此，正如移动端可以进行的某些操作不能在浏览器端进行，有些操作也是Appium可以但是Selenium却不行。因此，为了弥补这些不足，我们在传统Selenium客户端已有功能的基础上进行了相应的扩展，提供了一系列支持不同编程语言的Appium客户端。你可通过查看[Appium客户端列表](/docs/en/about-appium/appium-clients.md)页面，点击相应的链接进行下载。
```
Comment (lihuazhang) [inline docs/cn/about-appium/getting-started.md]: 你可通过查看[Appium客户端列表](/docs/en/about-appium/appium-clients.md)页面，

提供中文的地址

File: docs/cn/about-appium/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+node index.js
+```
+
+如果一切顺利，你将会看到Appium开始产生日志，最终屏幕上会弹出应用并且开始进行操作仿佛有一个无形的用户在进行点击
+
+### 接下来的内容
+
+上述内容中我们只是接触到了一些关于Appium方面的皮毛。查看下面所示文档更好的帮助你进行Appium的使用
+
+- [命令相关](https://appium.io/docs/en/commands/status/) - 学习可用的命令，如何将它们与特定的客户端进行使用等...
```
Comment (lihuazhang) [inline docs/cn/about-appium/getting-started.md]: 这里也应该用中文文档的地址

File: docs/cn/about-appium/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+appium
+```
+
+或者是点击Appium客户端页面上比较显眼的启动服务端按钮。
+
+Appium启动成功后将会向你展示一段简短的欢迎消息，其中包含了你目前所使用的Appium版本号和正在监听的端口号（默认使用`4723`）。该监听端口信息很重要，因为你需要确认测试客户端能通过该端口号顺利连接上Appium。如果你想要修改默认端口号，你可以在启动Appium时通过使用`-p`选项进行端口的设置（请确认已经查看过完整的[服务端参数列表](/docs/en/writing-running-appium/server-args.md)文档）
+
+### 运行你的第一个测试用例
+
+在该部分，我们将会运行一个简单的“Hello World”安卓测试用例。之所以选择安卓是因为它在所有平台上都能运行。同时，我们将使用[UiAutomator2 驱动](/docs/en/drivers/android-uiautomator2.md)，所以请确保你已经通读过相关文档并且系统已经具备所需的环境。本次将使用JavaScript作为开发语言，以便于不需要再使用到其他的依赖组件。
```
Comment (lihuazhang) [inline docs/cn/about-appium/getting-started.md]: 同时，我们将使用[UiAutomator2 驱动](/docs/en/drivers/android-uiautomator2.md) 一样要用中文文档

**Review by lihuazhang — CHANGES_REQUESTED** [review]

**Comment by XiufanJi** [comment]
多谢指正，后续会更加的仔细一点，前面提出问题的地方都已修改提交

**Review by lihuazhang — APPROVED** [review]

**Pull request merged**